control-plane/pkg/reconciler/broker/broker.go

```diff
@@ -572,8 +572,17 @@ func storeConfigMapAsStatusAnnotation(broker *eventing.Broker, cm *corev1.Config
 		broker.Status.Annotations = make(map[string]string, len(cm.Data))
 	}
 
+	keysToStore := map[string]bool{
```

Comment:
@basit9958 could you make this a top-level var, rather than creating this map every time this function is called? I.e. on line 67, just after the consts, could you add:

```go
var (
    keysToStore := ...
)
```